cutevariant/gui/vqleditor.py

```diff
@@ -112,20 +116,28 @@ def __init__(self):
         self.highlighter = VqlSyntaxHighlighter(self.text_edit.document())
         main_layout = QVBoxLayout()
         main_layout.addWidget(self.text_edit)
-        main_layout.setContentsMargins(0,0,0,0)
+        main_layout.setContentsMargins(0, 0, 0, 0)
         self.setLayout(main_layout)
         self.text_edit.textChanged.connect(self.changed)
```

Comment:
This signal should not connect to self.changed, but to a checkVQL() methods.

Reply:
I dont't really understand what it is supposed to do :p

Reply:
I will update this